Translated some pages into German (#85)
qmk/qmk.fm_old: _i18n/de/powered.md

---
title: "Die QMK Firmware Schutzmarke verwenden"
layout: qmk
permalink: /de/powered/
---
# Die QMK Schutzmarke

> "QMK Firmware" und das Logo sind von Jack Humbert registrierte Schutzmarken, um Tastaturen-Hersteller zu ermutigen zur Community beizutragen und der Community Unterstützung für Tastaturen mit QMK zu bieten.

Reviewer: ```suggestion
"QMK Firmware" und das Logo sind von Jack Humbert registrierte Schutzmarken, um Tastaturen-HerstellerInnen zu ermutigen zur Community beizutragen und der Community Unterstützung für Tastaturen mit QMK zu bieten.
```

> Jeder ist berechtigt und ermutigt das Logo, die Begriffe "QMK (Firmware)", "Powered by QMK (Firmware)" (oder ähnlich Formulierungen mit "running", "compatible with", etc.) verwenden. Dem wird vorausgesetzt, dass: 

Reviewer: ```suggestion
Alle sind berechtigt und ermutigt das Logo, die Begriffe "QMK (Firmware)", "Powered by QMK (Firmware)" (oder ähnlich Formulierungen mit "running", "compatible with", etc.) verwenden. Dem wird vorausgesetzt, dass: 
```

> 1. Nicht impliziert wird, dass QMK oder Beitragende die Tastatur befürworten / herstellen

Reviewer: ```suggestion
1. Nicht impliziert wird, dass QMK oder Mitwirkende die Tastatur befürworten oder herstellen
```

> 2. [Der Hersteller seine Konfigurationen der Tastatur zu qmk:master zeitlich nach Versand der Tastaturen beitragen](https://github.com/qmk/qmk_firmware/pulls/)

Reviewer: ```suggestion
2. [Der/Die HerstellerIn die Konfiguration der Tastatur zeitnah zum Release der Tastatur zu qmk:master beitragen](https://github.com/qmk/qmk_firmware/pulls/)
```

3. Auf der Produkt-Webseite auf die [QMK-Webseite](https://qmk.fm) oder das [GitHub-Repository](https://github.com/qmk/qmk_firmware) verlinkt wird

Bilder mit dem Logo und bevorzugten Formulierungen zur Verwendung auf PCBs, Ausdrucken, Webseiten, Stickers, Fan-Artikeln, etc. sind unten zu finden.

## Optionale Schritte

> 1. Einen Etikett / Tag (`<keyboard>-<version>`z.B. `planck-4.2`) von dem geliefertem Commit mit der Tastatur-Firmware in [qmk/qmk_firmware](https://github.com/qmk/qmk_firmware) erfragen.

Reviewer: ```suggestion
1. Beantragen Sie ein Etikett / Tag (`<keyboard>-<version>`z.B. `planck-4.2`) für das Commit mit dem die Tastatur in [qmk/qmk_firmware](https://github.com/qmk/qmk_firmware) ausgeliefert wird.
Üblicherweise wird für das Tag eine kompilierte .hex/.bin Datei auf der Release-Seite hinzugefügt.
```

2. Sich über [GPL Einhaltung](https://fsfe.org/activities/ftf/useful-tips-for-vendors.de.html) bewusst sein - Dabei hilft es ein Etikett / Tag zu haben.

> Wenn Ihr Projekt QMK als die standardmäßige und offizielle Firmware verwendet, würden wir es gerne zu der Liste auf unserer Webseite hinzufügen. Also [geben Sie bescheid, indem ein Pull Request auf qmk/qmk.fm](https://github.com/qmk/qmk.fm/pulls/) öffnen, sollte es noch nicht dort sein!

Reviewer: ```suggestion
Wenn Ihr Projekt QMK als die standardmäßige und offizielle Firmware verwendet, würden wir es gerne zu der Liste auf unserer Webseite hinzufügen. Also [geben Sie uns Bescheid, indem Sie ein Pull Request auf qmk/qmk.fm](https://github.com/qmk/qmk.fm/pulls/) eröffnen, sollte es noch nicht dort sein!
```

## Bilder

Dies sind die Badges in der Standard-Größe im SVG-Format (PNGs: [dunkel](/assets/images/badge-dark.png), [hell](/assets/images/badge-light.png)):

<style>
td {
    border: 0;
}
</style>

<table>
    <tr>
        <td><a href="/assets/images/badge-dark.svg"><img src="/assets/images/badge-dark.svg" alt="QMK Badge Dark" /></a></td>
        <td><a href="/assets/images/badge-light.svg"><img src="/assets/images/badge-light.svg" alt="QMK Badge Light" /></a></td>
    </tr>
</table>

> A smaller version of each (pngs: [dark](/assets/images/badge-small-dark.png), [light](/assets/images/badge-small-light.png)):

Reviewer: ```suggestion
 
```

Eine jeweils kleinere Version (PNGs: [dunkel](/assets/images/badge-small-dark.png), [hell](/assets/images/badge-small-light.png)):

<table>
    <tr>
        <td><a href="/assets/images/badge-small-dark.svg"><img src="/assets/images/badge-small-dark.svg" alt="QMK Badge Small Dark" /></a></td>
        <td><a href="/assets/images/badge-small-light.svg"><img src="/assets/images/badge-small-light.svg" alt="QMK Badge Small Light" /></a></td>
    </tr>
</table>

> And badges for compatible or community-supported keyboards (pngs: [dark](/assets/images/badge-community-dark.png), [light](/assets/images/badge-community-light.png)):

Reviewer: ```suggestion
 
```

Und Badges für kompatible oder *von-der-Community-unterstützte*n Tastaturen (PNGs:  [dunkel](/assets/images/badge-community-dark.png), [hell](/assets/images/badge-community-light.png)):

<table>
    <tr>
        <td><a href="/assets/images/badge-community-dark.svg"><img src="/assets/images/badge-community-dark.svg" alt="QMK Community Badge Dark" /></a></td>
        <td><a href="/assets/images/badge-community-light.svg"><img src="/assets/images/badge-community-light.svg" alt="QMK Community Badge Light" /></a></td>
    </tr>
</table>

> **Bitte verwenden Sie keine bearbeiteten Versionen dieser Bilder** - Wenn Sie etwas anderes verwenden möchten, [lassen Sie es uns wissen](https://github.com/qmk/qmk.fm/issues), und wir werden unser bestes tun Ihren Wünschen nachzukommen.

Reviewer: ```suggestion
**Bitte verwenden Sie keine bearbeiteten Versionen dieser Bilder** - Wenn Sie etwas anderes verwenden möchten, [lassen Sie es uns wissen](https://github.com/qmk/qmk.fm/issues), und wir werden unser Bestes tun Ihren Wünschen nachzukommen.
```
